github/PullRequest.py

```diff
@@ -508,30 +426,25 @@ def create_issue_comment(self, body):
 
     def create_review(
         self,
-        commit=github.GithubObject.NotSet,
-        body=github.GithubObject.NotSet,
-        event=github.GithubObject.NotSet,
-        comments=github.GithubObject.NotSet,
-    ):
+        commit: Opt[github.Commit.Commit] = NotSet,
+        body: Opt[str] = NotSet,
+        event: Opt[str] = NotSet,
+        comments: Opt[list] = NotSet,
```

Comment:
list of what type?

Reply:
it's a dict, should I add a TypedDict for this?